pkgs/by-name/bm/bmputil/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "bmputil";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "blackmagic-debug";
+    repo = "bmputil";
+    rev = "v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
`tag` is not an valid argument, `rev` is required:

```
error: function 'anonymous lambda' called without required argument 'rev'
```

Reply:
You need to rebase your branch on master. The tag argument is a fairly recent addition.